### DISCUSSION TITLE
Whale2 and Thank you

### DISCUSSION THREAD

**Discussion by ejhoness** [sn4k3/UVtools#929] [Q&A]
Hello, thank you very much for the excellent program, I use it in all resin prints, thank you very much. But besides thanking you, I would like information about the nova whale2, I happen to use linux ubuntu, and prusa slicer, and to send files through the adb terminal, so if you could provide me with a way to facilitate the sending, and information about uvtools config network for whale2. Furthermore, I wish you a long and prosperous life. Thank you.

![IMG-20240112-WA0002 fw_](https://github.com/user-attachments/assets/cc8b889b-f0e1-4447-94ec-c573fdf33dae)

![Captura de tela de 2024-09-19 13-57-11](https://github.com/user-attachments/assets/6276d48b-6bab-4dfa-9f0e-375215eca1a1)

![Captura de tela de 2024-09-19 14-12-09](https://github.com/user-attachments/assets/bdc5ac42-37cc-4e3d-bf47-14b8788082a4)

**Comment by sn4k3** [comment]
I don't know the protocol for the Whale2 network. You could try the defaults for Nova but I guess it won't work. Without a machine I can make some tests is very hard to include such.

About PrusaSlicer you can't use that upload function, it was made for Octoprint and Klipper. 

If you have a way to send files via terminal you can use UVtools, Settiings - General - "Send To - Process". That way you can trigger an command file File - Send To
